--- Makefile~	2008-04-09 21:53:41.000000000 +0200
+++ Makefile	2009-09-08 03:40:43.000000000 +0200
@@ -29,14 +29,14 @@
 
 
 CPPFLAGS += -Wall -I. -I/usr/X11R6/include `freetype-config --cflags` -fpic -DVERSION=\"$(VERSION)\" -D_REENTRANT -D_POSIX_PTHREAD_SEMANTICS -O2 
-LDFLAGS += -L$(PREFIX)/$(LIBDIR) -L/usr/X11R6/$(LIBDIR) `freetype-config --libs`
-LDLIBS +=
+LDFLAGS += -L$(PREFIX)/$(LIBDIR) -L/usr/X11R6/$(LIBDIR)
+LDLIBS += `freetype-config --libs`
 
 
 CLXCLIENT_SO = libclxclient.so
 CLXCLIENT_MAJ = $(CLXCLIENT_SO).$(MAJVERS)
 CLXCLIENT_MIN = $(CLXCLIENT_MAJ).$(MINVERS)
-CLXCLIENT_DEP = -lpthread -lXft -lX11
+CLXCLIENT_DEP = -lpthread -lXft -lX11 -lclthreads -lrt
 CLXCLIENT_O = xdisplay.o xresman.o xhandler.o xwindow.o xdraw.o \
 	button.o textip.o enumip.o menuwin.o scale.o slider.o scroll.o mclist.o meter.o
 CLXCLIENT_H = clxclient.h
